Dockerfile

```diff
@@ -11,8 +11,10 @@ RUN apk add --update --no-cache chromium chromium-chromedriver
 RUN adduser -D auto-southwest-check-in -h /app
 USER auto-southwest-check-in
 
+ENV PATH=/app/.local/bin:$PATH
+
 COPY requirements.txt requirements.txt
-RUN pip3 install --upgrade pip && pip3 install --no-cache-dir -r requirements.txt
+RUN python -m pip install --upgrade pip && pip3 install --upgrade pip && pip3 install --no-cache-dir -r requirements.txt
```

Comment:
The `pip3 install --upgrade pip` command already upgrades the `pip` module, so we shouldn't need this. I also verified in the Docker container that `pip` and `pip3` are the same binaries.